.gitignore

```diff
@@ -34,3 +34,5 @@ wheels/
 *.egg-info/
 .installed.cfg
 *.egg
+.vim
+.nvimrc
```

Comment:
Heh, Didn't know you used Vim. I've tried a few times, Never succeeded.

Reply:
I literally cannot use any other text editor now lol.